elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideStandaloneSettings.java

```diff
@@ -234,6 +234,15 @@ default Integer getAsyncQueryCleanupDays() {
         return 7;
     }
 
+    /**
+     * Number of seconds  between async query cancellation checks.
+     *
+     * @return Default: 300
+     */
+    default Integer getAsyncQueryCancelDelaySeconds() {
```

Comment:
i think getAsyncQueryCancelIntervalSeconds sounds better as Delay could be taken as differently.